dblp-aliases.csv

```diff
@@ -2193,3 +2193,4 @@ Muhammad Zubair Shafiq,Zubair Shafiq
 Oscar Cánovas Reverte,Óscar Cánovas Reverte
 Óscar Cánovas,Óscar Cánovas Reverte
 Oznur Tastan,Öznur Tastan
+Kui Ren,Kui Ren 0001

```

Comment:
Please delete this. The main entry should be the DBLP entry (that is, Kui Ren 0001).